### DISCUSSION TITLE
Welcome to Remotely Discussions!

### DISCUSSION THREAD

**Discussion by bitbound** [immense/Remotely#183] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stevesobol** [comment]
I'll start...

I'm Steve. IT professional for over 25 years, developer for about 22.5 of those years. I am customizing Remotely for my own use and hope to contribute some of my additions/mods back to the community. Probably will start with SSO. Shouldn't be too difficult, based on what I've seen of the source code.

**Comment by swilkey** [comment]
I'm going to try and start a trend here with names and experience...
I'm Steve IT professional for over 25 years, by training I'm a programmer but that was before object oriented programming began, by experience I've spent most of my career in infrastructure.  I'm working for an NGO (not for profit) that's been in Afghanistan for over 50 years.  In a country where internet connections are slow and computer knowledge is basic, having a tool like remotely that we can use to support staff in remote areas and that can cope with poor quality internet connections is really important to us.  We found out about remotely just as COVID began hitting the world and it has been a great help to us since then.

**Comment by lincoln131** [comment]
Hello! I'm Charles, and I've been in IT for a little over 17 years. I've been working in local government most of that time. I just found out about Remotely from Reddit, and I'm impressed!
